tools/segmetrics/run-segmetrics.py

```diff
@@ -128,6 +128,6 @@ def is_image_filepath(filepath):
 
     # write results
     with open(args.results, 'w', newline='') as fout:
-        csv_writer = csv.writer(fout, delimiter=';', quotechar='|', quoting=csv.QUOTE_MINIMAL)
+        csv_writer = csv.writer(fout, delimiter=',', quotechar='"', quoting=csv.QUOTE_MINIMAL)

```

Comment:
in general we recommend to use tabular files - this is Galaxy can render tabular files currently better and more tools support tabular than csv

Reply:
ok thanks, I will have a look into tabular files later and consider using them instead of CSV

Reply:
@bgruening do you mean this here? https://usegalaxy.org/static/formatHelp.html#tab

> One or more columns of text data separated by tabs.

isn't this simply CSV with tabulator delimiters?